AGPs/AGP-Comprehensive-Aragon-Bug-Bounty-Program.md

```diff
@@ -0,0 +1,56 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Bug Bounty Program
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Bug Bounty Program
+
+## Description of desired Association policy change
+
+This proposal is to create a comprehensive [Bug Bounty Program](https://wiki.aragon.org/dev/bug_bounty/) for all funded Aragon applications and tooling. 
```

Comment:
```suggestion
This proposal is to create a comprehensive [Bug Bounty Program](https://wiki.aragon.org/association/security/bug_bounty/) for all funded Aragon applications and tooling. 
```

Reply:
Thanks!

Reply:
Question: if I click `Resolve conversation` will it automatically merge your suggestions in?

Reply:
@burrrata it appears not, due to the [issue](https://github.com/aragon/AGPs/pull/100#issuecomment-540270809) mentioned in the other thread. I will create a new PR so it's easier to make edits on these files later today.